src/App.tsx

```diff
@@ -3,21 +3,36 @@
  */
 
 import React from 'react';
-import Menu from './components/Menu';
 import './App.css';
-import SectraTheme from './components/SectraTheme';
+import { BoxPlotChart, BarChart } from './components/Charts';
 
 /**
  * Top level component.
  *
  * @returns top level component
  */
+
+/**
+ * App funct
+ *
+ * @returns The app :)
+ */
 function App(): JSX.Element {
   return (
     <div className="App mui-theme">
-      <SectraTheme>
-        <Menu />
```

Comment:
shouldn't this still be in the code? Now everything on the site is removed except for the graphs

Reply:
Would this not be solved in the mearge with main? We removed it while testing and wanted a more "decluttered" App.tsx.

Reply:
Ah right so the idea with merging main into the branch before the PR gets accepted is that we fix any problems and fix layout and such. When this PR gets accepted this will be the new main so if we wanted to add more things there would have to be another PR. So I think the graphs and such should be added into the correct boxes and then we can review if the PR should be accepted

Reply:
Ok. Maybe we should wait with mergin 7 with main to after the other, current, PRs are merged?